### ISSUE TITLE
Agent that manages a RWO?

### ISSUE THREAD

**Issue by azaroth42** [hybox/models#40]
Need a predicate for holding institution. E.g. if some agent is the creator, and then donates the object/collection to an institution to look after, but still retains ownership ... what's the relationship between the object and the institution?

It's not the publisher (it could be locked away in a vault, this is the RWO collection, not the digital representation) or the rightsHolder (the creator is still the rights holder), nor a creator nor contributor ... so we're outside of DPLA MAP.  

marcrel has:
- Repository: An organization that hosts data or material culture objects and provides services to promote long term, consistent and shared use of those data or objects

Which seems pretty much what we want?

**Comment by anarchivist** [comment]
What is the assumption of the role of the holding institution? Physical custody? Legal ownership? Something else?

**Comment by azaroth42** [comment]
I think custody rather than ownership. The institution doesn't necessarily own the objects that it holds, but it does at least have custody of them.

**Comment by anarchivist** [comment]
A few (not closely considered) options. (also cc @guegueng): 
- `dct:provenance` (range: `dct:ProvenanceStatement`)
- `edm:currentLocation` (range: `edm:Place`)

**Comment by azaroth42** [comment]
I'm not keen on either, as there's no agency involved. A place does not look after an object nor does a Statement.

**Comment by anarchivist** [comment]
Agreed. How about [`dm2e:holdingInstitution`](http://onto.dm2e.eu/schemas/dm2e/holdingInstitution)?

**Comment by anarchivist** [comment]
(Looking at DM2E, we could also then add `dm2e:owner` if we had use cases for it.)

**Comment by azaroth42** [comment]
I'm happy with domain and range of `holdingInstitution`. We'd not want to use it on a pcdm:Object (as then it would become a ProvidedCHO) but that's okay.

Would we prefer DM2E or CRM? There's also:  crm:P50_has_current_keeper
Domain of Physical Thing, range of Actor

**Comment by anarchivist** [comment]
> Would we prefer DM2E or CRM?

Hmm... I lean towards DM2E (because it's opinionated in an EDM direction), but I could be flexible.

**Comment by azaroth42** [comment]
I'm cool with DM2E, and we already have an EDM/DPLA baseline so doesn't make sense to stray further than necessary.
